pkg/irreparable/irreparable.go

```diff
@@ -0,0 +1,174 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package irreparable
+
+import (
+	"context"
+	"time"
+
+	"go.uber.org/zap"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
```

Comment:
grpc should not be imported

Reply:
The codes are used in the "func (db *Database) validateAuth(ctx context.Context) error" kept similar to pointerdb and statdb. similar apikey validation adopted..So used the same error reporting

Reply:
DB shouldn't have authentication. `statdb` package is not really just a package, it was initially implemented as a service hence it needed the authentication parts.